<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="windowModality">
            <enum>Qt::NonModal</enum>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>819</width>
                <height>410</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Books</string>
        </property>
        <widget class="QTableView" name="books_QTableView">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>10</y>
                    <width>491</width>
                    <height>391</height>
                </rect>
            </property>
            <property name="contextMenuPolicy">
                <enum>Qt::ActionsContextMenu</enum>
            </property>
        </widget>
        <widget class="QGroupBox" name="groupBox">
            <property name="geometry">
                <rect>
                    <x>510</x>
                    <y>10</y>
                    <width>301</width>
                    <height>80</height>
                </rect>
            </property>
            <property name="title">
                <string>Search</string>
            </property>
            <widget class="QComboBox" name="search_comboBox">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>50</y>
                        <width>281</width>
                        <height>22</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="search_input">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>20</y>
                        <width>281</width>
                        <height>20</height>
                    </rect>
                </property>
            </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_5">
            <property name="geometry">
                <rect>
                    <x>510</x>
                    <y>90</y>
                    <width>301</width>
                    <height>51</height>
                </rect>
            </property>
            <property name="title">
                <string>Books</string>
            </property>
            <widget class="QPushButton" name="add_new_book_button">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>20</y>
                        <width>75</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Add New</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_7">
                <property name="geometry">
                    <rect>
                        <x>170</x>
                        <y>20</y>
                        <width>92</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Number of books:</string>
                </property>
            </widget>
            <widget class="QLabel" name="books_count">
                <property name="geometry">
                    <rect>
                        <x>270</x>
                        <y>20</y>
                        <width>18</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>000</string>
                </property>
            </widget>
        </widget>
        <widget class="QPushButton" name="close_button">
            <property name="geometry">
                <rect>
                    <x>740</x>
                    <y>380</y>
                    <width>75</width>
                    <height>23</height>
                </rect>
            </property>
            <property name="text">
                <string>Close</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
